pipelines/services/integration.go

```diff
@@ -164,14 +164,13 @@ func (is *IntegrationsService) createIntegration(integration IntegrationCreation
 		return -1, err
 	}
 	if err = errorutils.CheckResponseStatus(resp, http.StatusOK, http.StatusCreated); err != nil {
-		err := errorutils.GenerateResponseError(resp.Status, utils.IndentJson(body))
 		if resp.StatusCode == http.StatusConflict {
 			return -1, errorutils.CheckError(&IntegrationAlreadyExistsError{InnerError: err})
```

Comment:
I think we can now remove those error checks because the one you added to the CheckResponseStatus